content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -56,18 +56,21 @@ A cluster administrator creates a number of PVs. They carry the details of the r
 #### Dynamic
 When none of the static PVs the administrator created matches a user's `PersistentVolumeClaim`,
```

Comment:
I would like to move away from using the term "static" to describe manually created volumes. Can we change all references to "static" to something like "manually pre-created" or "pre-existing"?